docs/write_your_tiltfile.md

```diff
@@ -0,0 +1,79 @@
+# Write Your Tiltfile
+
+A Tiltfile can be broken down in to three things: Kubernetes YAML, Docker images and resources. In this doc we'll step through all three concepts so you can learn to write your own Tiltfiles with gusto!
+
+To start here's an illustrative example that uses most of the features of Tilt.
```

Comment:
I'm a bit hesitant about the use of "most" here. Maybe I've got skewed values, but I feel like it's already missing key features, and is likely to be missing more features in the future (like once we have some kind of group_by)